releasenotes/notes/discrete_signal_bounds_handling-e960ae879c409233.yaml

```diff
@@ -0,0 +1,14 @@
+---
+depreciated:
+  - |
+    The internal attribute `DiscreteSignal._samples` has been removed,
+    all methods should now be calling the public method `DiscreteSignal.samples`.
+
+other:
```

Comment:
Similarly here, no need to refer to `_samples`. The description also shouldn't refer to new private methods.

You could maybe say this more concisely:

```
The `DiscreteSignal.envelope` function has been changed so that if `t` is outside the interval defined by `start_time` and `start_time + dt * len(samples)` it will return `0.0`.
```